src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -162,187 +191,161 @@ When working to improve the user experience, it's best to think of the process a
 
-#### Measure website health with the CrUX dashboard and collected field data
+#### Analyze site performance with PageSpeed Insights
 
 <figure>
-  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/oF2PQELFtdICPN10aZge.png", alt="A screenshot of the CrUX dashboard. The dashboard breaks down LCP, FID, and CLS into desktop and mobile categories, with each category showing the distribution of values that lie within 'Good', 'Needs Improvement' and 'Poor' thresholds for the previous month.", width="800", height="837" %}
+  {% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/YEe3RQwgIWgQTHFV5sc0.png", alt="A screenshot of how PageSpeed Insights portrays CrUX data for a URL's Core Web Vitals. Each of the Core Web Vitals is displayed separately, while grouping each Core Web Vital in the 'Good', 'Needs Improvement', and 'Poor' thresholds for the last 28 days.", width="800", height="477" %}
 </figure>
 
-To get a snapshot of your website's performance, you can [create a CrUX Dashboard](https://developer.chrome.com/blog/chrome-ux-report-looker-studio-dashboard/) which reports the following:
+PageSpeed Insights displays the CrUX data covering the last 28 days of user experience data at the 75th percentile. This means that if 75% of user experiences meet the [threshold set for a given metric](/defining-core-web-vitals-thresholds/), then the experience is considered "good".
 
-- **Site overview**, which segments Core Web Vitals into desktop and mobile device types.
-- **Historical trend by metric type**, which is a distribution of metrics over time for each available monthly release of CrUX report data.
-- **User demographics**, which illustrates the distribution of page views across an entire origin for users in each demographic including device and [effective connection](https://developer.mozilla.org/docs/Glossary/Effective_connection_type) types.
+If you have a specific page in mind to look at the performance of, then use that. For an overall view of a site when you first start optimizing, you may wish to start with the home page as it is typically one of the most popular pages on most sites.
+
+Concentrate on the **what your real users are experiencing** section of PSI initially. You will see up to four views of the data: mobile and desktop for the URL entered and the whole origin. Compare these and see how they differ. Mobile is typically less performant than desktop since it is a more constrained device with a less stable network condition. If the URL and origin data are significantly different then try to understand why: home pages are often the first pages visited (so called landing page) so can be slower than the origin users take the full brunt of an uncached page. Subsequent pages will likely load faster as any shared assets will be cached, bringing down the aggregate origin-level data.
```

Comment:
```suggestion
Concentrate on the **what your real users are experiencing** section of PSI initially. You will see up to four views of the data: mobile and desktop for the URL entered and the whole origin. Compare these and see how they differ. Mobile is typically less performant than desktop since it is a more resource-constrained device operating under potentially less stable network conditions. If the URL and origin data are significantly different, then try to understand why: home pages are often the first pages visited (that is, a landing page) so can be slower than the origin users take the full brunt of an unprimed browser cache. Subsequent pages will likely load faster, as any shared assets will be cached, bringing down the aggregate origin-level data.
```